char/liberate/TEMPLATE/template_example.tcl

```diff
@@ -0,0 +1,44 @@
+# Liberate Template Example Tcl File - Sept 2008
+
+set lowSlew 0.14
+set highSlew 0.86
+
+# Set the maximum output transition time allowed
+set_var max_transition 1e-8
+
+set fp [open "celllist"]
+set cells [regexp -all -inline {\S+} [read $fp]]
+#set cells "INVX1"
+
+close $fp
+
+define_template -type delay \
+        -index_1        {0.1000 0.5000 1.2000 3.0000 4.0000 5.0000} \
+        -index_2        {0.0600 0.2400 0.4800 0.9000 1.2000 1.8000} \
+        delay_template
+
+define_template -type power \
+        -index_1        {0.1000 0.5000 1.2000 3.0000 4.0000 5.0000} \
+        -index_2        {0.0600 0.2400 0.4800 0.9000 1.2000 1.8000} \
+        power_template
+
+define_template -type constraint \
+        -index_1  {0.1  2.50  5.00} \
+        -index_2  {0.1  0.90  1.80} \
+        constraint_template
+
+set inputs  {A A0 A1 B C D CI Sel} 
```

Comment:
seems like the different between the `.template` and the non template version are those variables. Couldn't those bepassed as arguments, thru the environment or thru a separate configuration tcl file that you source?

Reply:
fixed